client/src/components/Ticketing/ticketingmanager/Season/components/SeasonInfo.tsx

```diff
@@ -121,6 +148,10 @@ const SeasonInfo = (props: SeasonProps) => {
       void handleCreateNewSeason(reqObject);
     } else {
       void handleUpdateSeason(reqObject);
+
+      if (props.onUpdateSeasonTicketType) {
```

Comment:
For consistency, can you destructure `onUpdateSeasonTicketType` from `props` at the top level of the component instead of using the dot notation here?

Reply:
Refactored